datamap/bloom/pom.xml

```diff
@@ -1,82 +0,0 @@
-<project xmlns="http://maven.apache.org/POM/4.0.0"
```

Comment:
I think now it is delete and add insted of move. we should use git move and git rename, so that it wont show as added , modified count

Reply:
because when I move bloom to index/bloom. 0 files are added and modifed

x-special/nautilus-clipboard
copy
recent:///eb75e249accc0787f48c636f5e4f795f